oak-auth-external/src/test/java/org/apache/jackrabbit/oak/spi/security/authentication/external/TestIdentityProvider.java

```diff
@@ -283,4 +285,20 @@ public ForeignExternalGroup() {
             return ImmutableList.of();
         }
     }
+    public static class UserWithSupplierProperties extends TestUser {
```

Comment:
trivial nitpicking: there should be an extra line separating the UserWithSupplierProperties from the previous class.